pkg/cvo/metrics.go

```diff
@@ -83,6 +85,10 @@ version for 'cluster', or empty for 'initial'.
 			Name: "cluster_installer",
 			Help: "Reports info about the installation process and, if applicable, the install tool.",
 		}, []string{"type", "version", "invoker"}),
+		clusterVersionOperatorRetrieving: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "cluster_version_operator_retrieving",
```

Comment:
Can this be `cluster_version_operator_update_retrieval` or some such?

Reply:
Or, I guess `cluster_version_operator_can_retrieve_updates` to match the 1 vs. 0 sense we're using for values.

Reply:
I chose it based on status.md, "...CVO is succesfully retrieving updates", but doesn't matter to me. It does appear that in other cases this name ends up being the camel-cased name used for the GaugeVec so this comment and https://github.com/openshift/cluster-version-operator/pull/357#discussion_r419700881 are related. Also see former comment for my reason for not wanting to using a negative named condition.

Reply:
06c550ff9f's `cluster_version_operator_update_retrieval_timestamp_seconds` works for me.